pnpm-lock.yaml

```diff
@@ -241,6 +241,9 @@ importers:
       '@storybook/addon-themes':
```

Comment:
Call `pnpm install` another time and push changes.

CI fails because lock file is old, it happens often.